dags/analyzer_helper/discord/discord_extract_raw_infos.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from analyzer_helper.discord.extract_raw_info_base import ExtractRawInfosBase
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class DiscordExtractRawInfos(ExtractRawInfosBase):
+    def __init__(self, platform_id: str):
+        """
+        Initializes the class with a specific platform identifier.
+
+        Parameters
+        ----------
+        platform_id : str
+            The identifier for the platform.
+        """
+        super().__init__(platform_id)
+        self.client = MongoSingleton.get_instance().client
+        self.db = self.client[self.get_platform_id()]
+        self.collection = self.db["rawinfos"]
```

Comment:
The `rawinfos` collection is under the `guild_id` database. 
For the processed data we're going to save the data under `platform_id` database.

So the `rawinfos` collection could be get via the `self.client[guild_id]["rawinfos"]` and `rawmemberactivities` collection that we're saving processed raw data within it would be `self.client[platform_id]["rawmemberactivities"]`